pkg/cli/build/build.go

```diff
@@ -126,6 +128,15 @@ func parseImageDefinition(configDir, definitionFile string) (*image.Definition,
 
 	imageDefinition, err := image.ParseDefinition(configData)
 	if err != nil {
+		if errors.Is(err, image.ErrorInvalidSchemaVersion) {
+			m := "Invalid schema version specified. This version of Edge Image Builder supports the following schema versions: %s"
+			msg := fmt.Sprintf(m, strings.Join(version.SupportedSchemaVersions, ", "))
+			return nil, &cmd.Error{
+				UserMessage: msg,
+				LogMessage:  msg,
```

Comment:
nit: I'd suggest using the actual error value here, similarly to what we do for other errors.